body/stretch_body/wacc.py

```diff
@@ -215,30 +179,110 @@ def pack_config(self,s,sidx):
             sidx += 1
```

Comment:
`pack_config` differs in P0 and P1. P1 introduces the "enable_sync_mode" config to Wacc. `pack_config` should move to the Protocol_Px classes.

Reply:
I've removed the enable_sync stuff from Pimu and Wacc